### PR TITLE
Replaced all instances of "zz" + model.NewId() with NewTestId()(#18194)

### PR DESCRIPTION
**Pull request by chitramdasgupta**
#### Summary

In storetest/thread_store.go, replaced all instances of ```"zz" + model.NewId()``` with ```NewTestId()```

#### Ticket Link

  Fixes https://github.com/mattermost/mattermost-server/issues/18194

### CODE REVIEW

File: store/storetest/thread_store.go [hunk 1]
```diff
@@ -54,26 +54,26 @@ func testThreadStorePopulation(t *testing.T, ss store.Store) {
 		o := model.Post{}
 		o.ChannelId = c.Id
 		o.UserId = u.Id
-		o.Message = "zz" + model.NewId() + "b"
+		o.Message = NewTestId() + "b"
```
Comment (agnivade) [inline store/storetest/thread_store.go:57]: Please remove the `b` as well here and everywhere else.

Comment (chitramdasgupta) [reply store/storetest/thread_store.go:57]: Can you please guide me how to go about it?
Should I create a new pull request?

Comment (chitramdasgupta) [reply store/storetest/thread_store.go:57]: Or should I commit the new changes to this pull request itself?

Comment (agnivade) [reply store/storetest/thread_store.go:57]: You can commit your new changes to this PR itself. Create a new commit and push it. The PR will update itself.

Comment (chitramdasgupta) [reply store/storetest/thread_store.go:57]: Ok thank you.

**Review by agnivade — APPROVED** [review]
Looks good!

**Review by isacikgoz — APPROVED** [review]
Nice 👍

**Pull request merged**